rca/enquire_to_study/views.py

```diff
@@ -18,22 +52,112 @@ def dispatch(self, *args, **kwargs):
         return super().dispatch(*args, **kwargs)
 
     def post_mailchimp(self):
```

Comment:
This method is called below but doesn't do anything, might it be worth raising `NotImplemented`, or add a comment about what needs to added?

Reply:
added a comment